packages/cli/src/lib/cmds/doctor.ts

```diff
@@ -0,0 +1,76 @@
+/*
+ * Copyright 2020 Google Inc. All Rights Reserved.
+ *
+ *  Licensed under the Apache License, Version 2.0 (the "License");
+ *  you may not use this file except in compliance with the License.
+ *  You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  Unless required by applicable law or agreed to in writing, software
+ *  distributed under the License is distributed on an "AS IS" BASIS,
+ *  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *  See the License for the specific language governing permissions and
+ *  limitations under the License.
+ */
+
+import {Log} from '@bubblewrap/core';
+import {join} from 'path';
+import {existsSync, promises as fsPromises} from 'fs';
+import {loadOrCreateConfig} from '../config';
+
+async function jdkDoctor(log: Log): Promise<boolean> {
+  const config = loadOrCreateConfig();
+  const jdkPath = (await config).jdkPath;
+  // Checks if the given path is a real path.
+  if (!existsSync(jdkPath)) {
+    log.error('The jdkPath doesn\'t exist, please run the' +
+    'following command to update it:\n' +
+    'bubblewrap updateConfig --jdkPath path/here. after thet run bubblewrap doctor again');
+    return false;
+  }
+  const file = await fsPromises.readFile(join(jdkPath, 'release'), 'utf-8');
+  if (!file) {
+    log.error('The jdkPath isn\'t correct, please run the' +
+    'following command to update it:' +
+    '\nbubblewrap updateConfig --jdkPath path/here, such that the folder "here" ' +
+    'contains the file "release.txt". after thet run bubblewrap doctor again');
+    return false;
+  };
+  // Checks if the jdk's version is 8 as needed.
+  if (file.indexOf('JAVA_VERSION="1.8') < 0) {
+    log.error('Wrong jdk version. You should download "OpenJDK' +
+    ' 8(LTS)" at the link below: \n' +
+    'https://adoptopenjdk.net/releases.html?variant=openjdk8&jvmVariant=hotspot');
+    return false;
+  }
+  return true;
+}
+
+async function androidSdkDoctor(log: Log): Promise<boolean> {
+  const config = loadOrCreateConfig();
+  const androidSdkPath = (await config).androidSdkPath;
+  // Checks if the given path is a real path.
+  if (!existsSync(androidSdkPath)) {
+    log.error('The androidSdkPath doesn\'t exist, please run the' +
```

Comment:
Maybe it would be nice to do this:

```
log.error('The androidSdkPath doesn't exist, please run the following command:');
log.info('    bubblewrap updateConfig --androidPath path/to/sdk');
log.error('Then run bubblewrap doctor again.');
```

That way the command to run is somewhat highlighted (well, it's white instead of red).

What do you think? (If it turns out looking bad, we won't do it.)

@andreban since you've been working on making the UI pretty.
```

Reply:
The thing about mixing `log.error` and `log.info` is that I think one goes to `stdout` and the other to `stderr`, which would lead to some weird behaviour if we redirect one of them to a file, for instance.

We are using the `colors` module for the `validator` already. Maybe something like:

```typescript
import {bold} from 'colors';
```
then
```typescript
log.error(`The androidSdkPath doesn't exist, please run the following command:
       ${colors.bold('bubblewrap updateConfig --androidPath path/to/sdk'})
Then run ${colors.bold('bubblewrap doctor') again.`);
```